### PR TITLE
Utilise user name of current user (current context) instead of hardcoded test user

### PR DESCRIPTION
**Pull request by tim-powell-eviden**

### CODE REVIEW

**Review by udaydefra — COMMENTED** [review]
Signout functionality is broken. Can you get the changes from main?

File: src/EPR.Calculator.Frontend/Views/Dashboard/Index.cshtml [hunk 1]
```diff
@@ -1,28 +1,20 @@
...
-
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
         <p></p>
     </div>
     <div class="govuk-grid-column-one-quarter govuk-body-s govuk-!-margin-top-4">
         <div class="account-info">
-            <span class="your-name" id="bannerUserName">@currentUser</span>
-            <a asp-controller="Account" asp-action="Signout" asp-area="MicrosoftIdentity" class="account-link">Sign out</a>
```
Comment (udaydefra) [inline src/EPR.Calculator.Frontend/Views/Dashboard/Index.cshtml:25]: This needs to be added back

Comment (tim-powell-eviden) [reply src/EPR.Calculator.Frontend/Views/Dashboard/Index.cshtml:25]: Done

**Review by udaydefra — DISMISSED** [review]
LGTM

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by udaydefra — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by davidjayakumarEviden — APPROVED** [review]

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**